includes/OutputPage.php

```diff
@@ -464,7 +464,7 @@ protected function filterModules( $modules, $position = null, $type = ResourceLo
 	public function getModules( $filter = false, $position = null, $param = 'mModules' ) {
 		// Wikia change - begin - @author macbre
 		// Load all ResourceLoader modules at the bottom of the page
-		// when the skin has the flag set (e.g. Venus)
+		// when the skin has the flag set
 		$skin = $this->getSkin();
 		if ( $skin instanceof WikiaSkin && !empty( $skin->pushRLModulesToBottom ) ) {
```

Comment:
@macbre do you know if anything uses pushRLModulesToBottom? 
Comment indicates that it was used only by Venus but we are not sure